### PR TITLE
InProcess

### PR DESCRIPTION
**Pull request by humhei**

### CODE REVIEW

**Comment by dsyme** [comment]
Interesting!  It would be great to unify these!

**Comment by humhei** [comment]
Continue work tomorrow

File: src/FsLive.Porta/ProcessCommandLine.fs [hunk 1]
```diff
@@ -2,21 +2,23 @@
 
 // F# PortaCode command processing (e.g. used by Fabulous.Cli)
```
Comment (humhei) [inline src/FsLive.Porta/ProcessCommandLine.fs:3]: @dsyme 
The porta changed codes is limited to ProcessCommandLine.fs

**Comment by humhei** [comment]
Ready for review

Is it the time to rename this repository to FsLive? @dsyme

**Comment by humhei** [comment]
The breaking change is [Multiple target frameworks](https://github.com/humhei/FCSWatch/pull/9/files#diff-69c7e1fb1ddc6743b2d3fd3749aaa64bR292) and [reference projects](https://github.com/humhei/FCSWatch/pull/9/files#diff-40ff2e5fc15d01cb644eb1128850e71fR159)
They all worked 
Do they make sense for PortaCodes?

E.g 
The source file change in refereced project will also trigger compiling
Every target framework will trigger one time

**Comment by humhei** [comment]
These days i try to add porta codes to current project in branch [add-porta](https://github.com/humhei/FCSWatch/tree/Add-porta)

At last step, I finally find that i may not need porta codes(
As it don't reload whole application
And it's difficult for me to re-registration excel-functions
)

But all the work should be merged because porta codes actually worked(Though i don't use it now)

Any way,
We should share the some runtime for further development (FcsWatch.Core)

So let's merge it

**Comment by humhei** [comment]
Close this support #12

**Pull request closed without merging**